ddtrace/contrib/pytest/_utils.py

```diff
@@ -130,17 +144,21 @@ def _pytest_marked_to_skip(item: pytest.Item) -> bool:
     if item.get_closest_marker("skip") is not None:
         return True
 
-    return any([True for marker in item.iter_markers(name="skipif") if marker.args[0] is True])
+    return any(marker.args[0] for marker in item.iter_markers(name="skipif"))
 
 
 def _is_test_unskippable(item: pytest.Item) -> bool:
     """Returns True if a test has a skipif marker with value false and reason ITR_UNSKIPPABLE_REASON"""
     return any(
-        [
-            True
-            for marker in item.iter_markers(name="skipif")
-            if marker.args[0] is False
-            and "reason" in marker.kwargs
-            and marker.kwargs["reason"] is ITR_UNSKIPPABLE_REASON
-        ]
+        (marker.args[0] is False and marker.kwargs.get("reason") is ITR_UNSKIPPABLE_REASON)
```

Comment:
@romainkomorndatadog Any reason to use `is` instead of `==`?

Reply:
Oh, wow, no, it should be `==` ... That's been wrong from the start.

Good thing Python does fun things like
```
>>> a = "datadog_itr_unskippable"
>>> b = "datadog_itr_unskippable"
>>> a is b
True
>>> b = "datadog_itr_unskippable_2"
>>> a is b
False
>>> b = "datadog_itr_unskippable"
>>> a is b
True
```

Reply:
It's actually pretty important to use `==` here instead of `is`, because `==` tests for equality and `is` tests for identity (are these two the same object).